packages/salesforcedx-webview-ui/config/webpack.config.dev.js

```diff
@@ -0,0 +1,266 @@
+              limit: 10000,
+              name: 'media/[name].[ext]'
+            }
+          },
+          {
+            test: /\.(js|jsx|mjs)$/,
+            include: paths.appSrc,
+            loader: require.resolve('babel-loader'),
+            options: {
+              compact: true
+            }
+          },
+
+          // Compile .tsx?
+          {
+            test: /\.(ts|tsx)$/,
+            include: paths.appSrc,
+            use: [
+              {
+                loader: require.resolve('ts-loader'),
+                options: {
+                  // disable type checker - we will use it in fork plugin
+                  transpileOnly: true
+                }
+              }
+            ]
+          },
+          // "postcss" loader applies autoprefixer to our CSS.
+          // "css" loader resolves paths in CSS and adds assets as dependencies.
+          // "style" loader turns CSS into JS modules that inject <style> tags.
+          // In production, we use a plugin to extract that CSS to a file, but
+          // in development "style" loader enables hot editing of CSS.
+          {
+            test: /\.css$/,
+            use: [
+              require.resolve('style-loader'),
+              {
+                loader: require.resolve('css-loader'),
+                options: {
+                  importLoaders: 1
+                }
+              },
+              {
+                loader: require.resolve('postcss-loader'),
+                options: {
+                  // Necessary for external CSS imports to work
+                  // https://github.com/facebookincubator/create-react-app/issues/2677
+                  ident: 'postcss',
+                  plugins: () => [
+                    require('postcss-flexbugs-fixes'),
+                    autoprefixer({
+                      browsers: [
+                        '>1%',
+                        'last 4 versions',
+                        'Firefox ESR',
+                        'not ie < 9' // React doesn't support IE8 anyway
+                      ],
+                      flexbox: 'no-2009'
+                    })
+                  ]
+                }
+              }
+            ]
+          },
+          // "file" loader makes sure those assets get served by WebpackDevServer.
+          // When you `import` an asset, you get its (virtual) filename.
+          // In production, they would get copied to the `build` folder.
+          // This loader doesn't use a "test" so it will catch all modules
+          // that fall through the other loaders.
+          {
+            // Exclude `js` files to keep "css" loader working as it injects
+            // its runtime that would otherwise processed through "file" loader.
+            // Also exclude `html` and `json` extensions so they get processed
+            // by webpacks internal loaders.
+            exclude: [/\.(js|jsx|mjs)$/, /\.html$/, /\.json$/],
+            loader: require.resolve('file-loader'),
```

Comment:
Did you try something like 
{ test: /\.(jpe?g|png|gif|svg|woff|eot|ttf)$/i, loader: 'file-loader?name=/public/[name].[ext]' }
? I have it working like that in another project but it might be because I'm on a more recent version of file-loader.

Reply:
Will try. I wasn't sure if you could pack svg|woff|eot|ttf. That would help.

Reply:
I might be misunderstanding here but this should be OK because this is the _last_ branch of the _oneOf_ branch. That means we shouldn't need to add more tests. In fact, this test is the `not` of the other branches. 

So, we are already serving the svg, woff, eot, and ttf files. For jpeg, png, gif we actually convert embed them as data URLS to avoid any requests. See https://github.com/forcedotcom/salesforcedx-vscode/blob/e84a5178d55e911c2ba489598d514d88ad039132/packages/salesforcedx-webview-ui/config/webpack.config.dev.js#L121 and https://webpack.js.org/loaders/url-loader/

What I've done is also add woff|eot|ttf so that they get embedded.

I verified that we must be serving them properly because blueprint.js uses some custom font for the icons and we are seeing those icons in the manifest editor.